### PR TITLE
Push for 2.12

### PR DESCRIPTION
**Pull request by reactormonk**

### CODE REVIEW

**Comment by timperrett** [comment]
@reactormonk nice :-)

File: src/main/scala/RigPlugin.scala [hunk 1]
```diff
@@ -133,7 +133,7 @@ object common {
           "-Xlint:package-object-classes",
           "-Xlint:unsound-match",
           "-Xlint:stars-align"
-        )
+        ) ++ (if(n >= 12) {Seq("-Ypartial-unification")} else Seq())
```
Comment (timperrett) [inline src/main/scala/RigPlugin.scala]: Could we make this the same case match style as earlier lines? Would like to be consistent.

**Comment by timperrett** [comment]
@reactormonk looks good overall, just that one comment for style. Can we also bump the feature version of the plugin?

**Comment by reactormonk** [comment]
Is feature minor or major?

**Comment by timperrett** [comment]
@reactormonk oh sorry :-) The versioning scheme is 1.2.3 where 1 is major, 2 is feature and 3 is patch. Thanks for rolling up your sleeves to do this!

**Comment by reactormonk** [comment]
Not sure how your build farm picks it up. Considered https://github.com/dwijnand/sbt-dynver ?

**Comment by timperrett** [comment]
@reactormonk it will automatically build and release it once merged :-) We release sbt-rig with sbt-rig, so it automatically handles versioning

**Comment by reactormonk** [comment]
Right, oops.

**Pull request merged**